src/styles/utils/_variables.scss

```
@@ -0,0 +1,12 @@
+$gap-cell: 1px;
+$size-cell: 100px;
+$padding-calendar: 10px;
+$widht-calendar: 7 * $size-cell + 6 * $gap-cell + 2 * $padding-calendar;
```

what do 7 and 6 means? let's create a variable for it. For example $days-in-week
```suggestion
$widht-calendar: $days-in-week * $size-cell + ($days-in-week - 1) * $gap-cell + 2 * $padding-calendar;
```

---

There are values that are not changeable, such as:  number of days in a week,  "number Pi", etc. I don't see the point of setting them as variables. But if the customer or mentor wants it that way, then with pleasure)